appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/GalleryPage.java

```diff
@@ -931,8 +936,13 @@ public void onSuccess(Long id) {
                           submitReport.setEnabled(false);
                         }
                     };
-                  Ode.getInstance().getGalleryService().addAppReport(app, reportText.getText(),
-                      reportClickCallback);
+                  if (reportText.getText().replaceAll(" ", "").length() > 0){
```

Comment:
@josmas Yes that is what its doing, i'll replace it with trim() and isEmpty(). Thanks